lib/ansible/modules/cloud/vmware/vmware_dvswitch.py

```diff
@@ -35,6 +35,10 @@
         description:
             - The name of the switch to create or remove
         required: True
+    switch_version:
+        description:
+            - The version of the switch to create. Needed if you have a vcenter version > ESXi version to join DVS. If not specified version=version of vcenter
+        required: False
```

Comment:
add 'version_added: 2.5' here.

Reply:
Thank you Akasurde. I hope it will be ok now.